client_instantiate.go

```diff
@@ -0,0 +1,43 @@
+package sdk
+
+import (
+	"errors"
+	"fmt"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (c *Client) Instantiate(codeID uint64, label, instantiateMsg string, funds []sdktypes.Coin) (string, error) {
+	if instantiateMsg == "" {
+		return "", errors.New("message code is empty")
+	}
+
+	if label == "" {
+		return "", errors.New("label is empty")
+	}
+
+	if len(funds) == 0 {
```

Comment:
funds can be empty. Let's give this opportunity

Reply:
done